packages/transcription/src/index.ts

```diff
@@ -71,7 +72,7 @@ export const createTranscriptionProvider
           })
         }
 
-        worker.addEventListener('message', (event: MessageEvent<WorkerMessageEvent>) => {
+        worker?.addEventListener('message', (event: MessageEvent<WorkerMessageEvent>) => {
```

Comment:
Would `worker` be mandatory here? If so, should we _:panic:_?

Reply:
Yes. We can use `!` to tell TypeScript explicitly.